target/linux/brcm63xx/dts/dsl-2750u-c1.dts

```diff
@@ -0,0 +1,101 @@
+/dts-v1/;
```

Comment:
@naseef Please can you consider `SPDX-License-Identifier: GPL-2.0-or-later OR MIT` as in [DTS Checklist](https://openwrt.org/submitting-patches#dts_checklist) ?
> Having MIT as alternative is sometimes preferred by upstream maintainers
and allows sharing that simple code with other projects. We don't really
want multiple DTS versions for the same device.